core/src/test/java/com/linecorp/armeria/common/stream/ByteStreamMessageOutputStreamTest.java

```diff
@@ -160,6 +162,7 @@ void writeAfterStreamClosed() throws InterruptedException {
                     if (i < 2) {
                         os.write(i);
```

Comment:
`os.write()` was raising an exception after `.then(byteStreamMessage::abort)` is called below.
https://github.com/line/armeria/blob/179d9a507456552aafffbeb2854ef52b2687390c/core/src/main/java/com/linecorp/armeria/common/stream/ByteStreamMessageOutputStream.java#L169